source/reference/mongosync-behavior.txt

```diff
@@ -296,6 +147,44 @@ Read Preference
 connecting to the source and destination clusters. For more information,
 see :ref:`connections-read-preference`. 
 
+Destination Cluster Changes
+---------------------------
+
+Consistency
+~~~~~~~~~~~
+
+``mongosync`` supports :term:`eventual consistency` on the destination
+cluster. Read Consistency is not guaranteed on the destination cluster,
+the source and destination clusters may differ at a given point in
+time. However, if new writes are halted on the source cluster, the
+source and destination clusters will eventually hold the same data.
+
+Individual writes may be reordered or combined as they are relayed from
+source to destination. For a given document, the total number of writes
+may differ between source and destination.
+
+Transactions may appear non-atomically on the destination cluster.
+Retryable writes may not be retryable on the destination cluster.
+
+Profiling
+~~~~~~~~~
+
+If profiling is enabled on a source database, MongoDB creates a special
+collection named ``<db>.system.profile``. After synchronization is
+complete, {+c2c-product-name+} will not drop the
+``<db>.system.profile`` collection from the destination even if the
+source database is dropped at a later time. The ``<db>.system.profile``
+collection will not change the accuracy of user data on the
+destination.
+
+Views
+~~~~~
+
+If a database with views is dropped on the source, the destination may
+show an empty ``system.views`` collection in that database. The empty
+``system.views`` collection will not change the accuracy of user
+data on the destination.
+
 .. _c2c-capped-collections:
 
 Capped Collections
```

Comment:
Since capped collections and views have information that has to do with both source and destination clusters, how about a "Collection Considerations" section?

Reply:
Alternatively, "Capped Collections" can go in the collection considerations alone. I'm just hesitant to put it under Destination Cluster Changes since the info in this section has to do with both the source and the destination

Reply:
Discussed in slack -- creating "Cluster and Collection Types" heading.